spec/v3.spec.json

```diff
@@ -67,8 +66,11 @@
     "buildDependencies": {
       "title": "Build Dependencies",
       "type": "object",
+      "propertyNames": {
+        "$ref": "#/definitions/PackageName"
+	  },
       "patternProperties": {
-        "^[a-z][-a-z0-9]{0,255}$": {
+        "": {
```

Comment:
This must be JSON-schema syntax that I'm not familiar with.  Am I correct that the previous `"propertyNames"` definition gets applied to the keys for `"patternProperties"`?

Reply:
Yup, it was added in a [newer version of jsonschema](https://json-schema.org/understanding-json-schema/reference/object.html#property-names).